Platform/Google/msm8998/AcpiTables/walleye/dsdt_common.asl

```diff
@@ -0,0 +1,137 @@
+// To enable SOC revision based run time differentiation, uncomment following line
+// and uncomment SSID method in ABD device. The original string is artificailly set as 
+// 16 characters, so there is enough room to hold SOC revision string. 
+// To adjust the number, the MAX_SOCID_LEN macro as defined in ABD device.h should be
+// adjusted at the same time. 
+
+Name (SOID, 0xffffffff)          // Holds the Chip Id
+Name (SIDS, "899800000000000")   // Holds the Chip ID translated to a string
+Name (SIDV, 0xffffffff)          // Holds the Chip Version as (major<<16)|(minor&0xffff)
+Name (SVMJ, 0xffff)              // Holds the major Chip Version
+Name (SVMI, 0xffff)              // Holds the minor Chip Version
+Name (SDFE, 0xffff)              // Holds the Chip Family enum
+Name (SFES, "899800000000000")   // Holds the Chip Family translated to a string
+Name (SIDM, 0xfffffffff)         // Holds the Modem Support bit field
+Name (SOSN, 0xaaaaaaaabbbbbbbb)  // Holds the Chip Serial Number
+Name (PLST, 0xffffffff)  		 // Holds the Device platform subtype
+Name (RMTB, 0xaaaaaaaa)
+Name (RMTX, 0xbbbbbbbb)
+Name (RFMB, 0xcccccccc)
+Name (RFMS, 0xdddddddd)
+Name (RFAB, 0xeeeeeeee)
+Name (RFAS, 0x77777777)
+Name (PRP0, 0x0)                // Holds the enable/disable flag for PCIe
+Name (WLEN, One)                // Holds the enable/disable flag for WLAN
+
+Device (HAL0)
+{
+    Name (_HID, "QCOM0009")  // _HID: Hardware ID
+    Name (_UID, Zero)  // _UID: Unique ID
+    Alias (\_SB.PSUB, _SUB)
+}
+
+//
+// Storage - UFS/SD 
+//
+Include("ufs.asl")
```

Comment:
* Does not disable ufs cause problems?

Reply:
I never tried booting windows tbh i will disable it and push again